docs/managed-victoriametrics/alertmanager-setup-for-deployment.md

```diff
@@ -1,19 +1,27 @@
 ---
-title: Alertmanager and VMAlert configuration for Deployment
+sort: 4
+weight: 4
+title: Alertmanager and VMAlert configuration for Managed VictoriaMetrics deployment
+menu:
+  docs:
+    parent: "managed"
+    weight: 4
+aliases:
+  - /managed-victoriametrics/alertmanager-setup-for-deployment.html
 ---
 
 ## Alerting stack configuration and Managed VictoriaMetrics
```

Comment:
Should we update the overivew page and ADD information about these features there?
or quick start ?

Reply:
Good idea! Will do it 👍

Reply:
Done